_includes/block/scripts.html

```diff
@@ -13,8 +13,12 @@
 </script>
 {% endif %}
 
+{% if page.widgets contains 'addthis' or layout.widgets contains 'addthis' %}
+<script async src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5b039a09ed801a5d"></script>
```

Comment:
for content sharing, can we generate the share links? something like this: https://github.com/gergelyke/blog/blob/261a33aecb60ed3c9d93ab0405625f45f1dea65a/themes/cactus-light/layout/_partial/post/share.ejs#L3

so we wouldn't have to add another JS library to the page

Reply:
That could be nice if someone wants to take the time to develop that feature and submit a PR. For now, I feel this asynchronously-loaded widget is a nice first implementation of this new feature.

Reply:
we had that, it got removed: https://github.com/godaddy/godaddy.github.io/commit/6fe4a3791828850b21c00385c0c67d4e07587863#diff-663f387b6a1a407ab38de055a12bc7c8L22